### PR TITLE
5. Configurando las pruebas con TypeScript

### PR DESCRIPTION
**Pull request by cardel**
image

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (kliver98) [inline cypress/support/commands.ts]: remember this and e2e.js as well google.cy.js have to be renamed __not to create another file__
Read again the step please https://github.com/holgiosalos/cypress-training-doc#5-configurando-las-pruebas-con-typescript

File: cypress/support/e2e.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (kliver98) [inline cypress/support/e2e.ts]: End of line, add an enter after this line

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (kliver98) [inline cypress/support/commands.ts]: End of line, add an enter after this line

File: cypress/support/commands.ts (hunk 1 above)
Comment (cardel) [reply cypress/support/commands.ts]: Fixed

File: cypress/support/e2e.ts (hunk 2 above)
Comment (cardel) [reply cypress/support/e2e.ts]: Fixed

File: cypress/support/commands.ts (hunk 3 above)
Comment (cardel) [reply cypress/support/commands.ts]: Fixed

**Comment by cardel** [comment]
I fixed the repository, I had a problem with the mergen in my local repository, for this reason the change from js to ts has not been visible.

**Review by kliver98 — APPROVED** [review]

**Pull request merged**